controller/instance_manager_controller.go

```diff
@@ -547,15 +543,21 @@ func (imc *InstanceManagerController) syncLogSettingsToInstanceManagerPod(im *lo
 		}
 
 		switch settingName {
-		case types.SettingNameV2DataEngineLogLevel:
-			err = client.LogSetLevel(longhorn.DataEngineTypeV2, "spdk_tgt", setting.Value)
+		case types.SettingNameLogLevel:
+			if types.IsDataEngineV1(im.Spec.DataEngine) {
+				err = client.LogSetLevel(longhorn.DataEngineTypeV1, "", setting.Value)
+			} else {
+				err = client.LogSetLevel(longhorn.DataEngineTypeV2, "", setting.Value)
+			}
```

Comment:
What about other components like UI, csi plugin, csi side car pod as mentioned at https://github.com/longhorn/longhorn/issues/6702#issuecomment-2445030144

Reply:
That would be a separate PR, although less critical, since they don't log as heavily.  It shouldn't gate this change.